File: day 44/classwork/code/codewars3.py

"""https://www.codewars.com/kata/54c27a33fb7da0db0100040e/train/python"""

# You're a square!

def is_square(n):
    if n < 0:
        return False
    num = int(n ** 0.5)
    return num * num == n

File: day 080/homework/code/codewars3.js

// https://www.codewars.com/kata/5842df8ccbd22792a4000245/train/javascript

// Write Number in Expanded Form

function expandedForm(num) {
  return String(num)
  .split("")
  .map((digit, index, arr)=>
    digit+ "0".repeat(arr.length - index-1)  
  )
  .filter(x => !x.startsWith("0"))
  .join(" + ");
}